tests/e2e/test_subshell.py

```
@@ -1,3 +1,5 @@
+from webbrowser import get
```

Ruff

`webbrowser.get` imported but unused

Remove unused import: `webbrowser.get`

(F401)